accounts-cluster-bench/src/main.rs

```diff
@@ -340,6 +343,35 @@ fn run_rpc_bench_loop(
             break;
         }
         match rpc_bench {
+            RpcBench::GetAccountInfo => {
+                let mut rpc_time = Measure::start("rpc-get-account-info");
```

Comment:
why are we timing the setup?

Reply:
We're just super dumb.

ty for the catch.